spark/spark-scala-parent/src/main/scala/org/apache/zeppelin/spark/BaseSparkScalaInterpreter.scala

```diff
@@ -164,6 +164,8 @@ abstract class BaseSparkScalaInterpreter(val conf: SparkConf,
     if (BaseSparkScalaInterpreter.sessionNum.decrementAndGet() == 0) {
       if (sc != null) {
         sc.stop()
+      }else {
+        System.exit(1)
```

Comment:
Could you add one line of logging to illustrate that spark interpreter is exited before `SparkContext` created

Reply:
I now think system.exit(1) is redundant. Because the sparkSubmit and remoteinterpreterServer processes are in the same JVM, only the remoteinterpreterServer can exit normally.

Reply:
@MOBIN-F What do you mean ? Do you mean system.exit could not resolve this issue ?